### DISCUSSION TITLE
list.groupByYear vs term.groupByYear

### DISCUSSION THREAD

**Discussion by JVimes** [nunocoracao/blowfish#749] [General]
For `http://localhost:1313/tags/shortcodes/` from exampleSite, I enable year grouping by editing params.toml and setting groupByYear in the `[list]` section. I expected that from the `[term]` section. I think /tags/shortcodes/ is a "term", right? Is this a bug? Thank you.

**Comment by nunocoracao** [comment]
No that should work - and I tested it in the last version and it worked.
Could you please try again and confirm that you are not seeing something like image below?
<img width="928" alt="Screenshot 2023-06-04 at 21 24 18" src="https://github.com/nunocoracao/blowfish/assets/3803196/075039c6-4a9c-4329-a44a-55f53e6f840c">

**Reply by JVimes** [reply]
Latest commit 14ceca7ca11492611242901d3ad7de4df7af54d3 still reacts to _list_.groupByYear instead of _term_.groupByYear for me. I think 'shortcodes' is a [term](https://gohugo.io/content-management/taxonomies/#definitions).

https://github.com/nunocoracao/blowfish/assets/6148201/376b88bb-f223-41b2-97f3-de6b22c4cd08

**Reply by nunocoracao** [reply]
No, it's a list of articles not a list of terms.

**Reply by JVimes** [reply]
Ah, makes sense. Thank you!
